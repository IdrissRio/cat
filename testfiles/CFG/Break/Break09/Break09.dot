digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[22]"; "Opt[25]"; "List[26]"; "List[27]"; "InstanceInitialization[6]"; "InstanceInitialization[14]"}
{ rank = same; "List[23]"; "StaticInitializer[28]"; "Entry[7]"; "Exit[8]"; "Entry[15]"; "Exit[16]"}
{ rank = same; "Modifier[24]"; "Block[29]"}
{ rank = same; "List[30]"}
{ rank = same; "VarDeclStmt[31]"; "SwitchStmt[40]"; "VarDeclStmt[89]"}
{ rank = same; "Modifiers[32]"; "PrimitiveTypeAccess[34]"; "List[35]"; "VarAccess[41]"; "Block[42]"; "Modifiers[90]"; "PrimitiveTypeAccess[92]"; "List[93]"}
{ rank = same; "List[33]"; "VariableDeclarator[36]"; "List[43]"; "List[91]"; "VariableDeclarator[94]"}
{ rank = same; "List[37]"; "Opt[38]"; "ConstCase[44]"; "VarDeclStmt[46]"; "BreakStmt[54]"; "ConstCase[57]"; "VarDeclStmt[59]"; "ConstCase[67]"; "VarDeclStmt[69]"; "BreakStmt[77]"; "DefaultCase[80]"; "VarDeclStmt[81]"; "List[95]"; "Opt[96]"}
{ rank = same; "IntegerLiteral[39]"; "IntegerLiteral[45]"; "Modifiers[47]"; "PrimitiveTypeAccess[49]"; "List[50]"; "List[56]"; "UncheckedExceptions[55]"; "IntegerLiteral[58]"; "Modifiers[60]"; "PrimitiveTypeAccess[62]"; "List[63]"; "IntegerLiteral[68]"; "Modifiers[70]"; "PrimitiveTypeAccess[72]"; "List[73]"; "List[79]"; "UncheckedExceptions[78]"; "Modifiers[82]"; "PrimitiveTypeAccess[84]"; "List[85]"}
{ rank = same; "List[48]"; "VariableDeclarator[51]"; "List[61]"; "VariableDeclarator[64]"; "List[71]"; "VariableDeclarator[74]"; "List[83]"; "VariableDeclarator[86]"}
{ rank = same; "List[52]"; "Opt[53]"; "List[65]"; "Opt[66]"; "List[75]"; "Opt[76]"; "List[87]"; "Opt[88]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Break09 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[22]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[23]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[24]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[25]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[26]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[27]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"StaticInitializer[28]" [label="StaticInitializer\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[29]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[30]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[31]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[32]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[33]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[34]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[35]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[36]" [label="VariableDeclarator\n < i >", fillcolor=white   style=filled  ]
"List[37]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[38]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[39]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"SwitchStmt[40]" [label="SwitchStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarAccess[41]" [label="VarAccess\n < i >", fillcolor=white   style=filled  ]
"Block[42]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[43]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ConstCase[44]" [label="ConstCase\n < 1 >", fillcolor=white   style=filled  ]
"IntegerLiteral[45]" [label="IntegerLiteral\n < 1 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[46]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[47]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[48]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[49]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[50]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[51]" [label="VariableDeclarator\n < c >", fillcolor=white   style=filled  ]
"List[52]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[53]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"BreakStmt[54]" [label="BreakStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[56]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[55]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ConstCase[57]" [label="ConstCase\n < 2 >", fillcolor=white   style=filled  ]
"IntegerLiteral[58]" [label="IntegerLiteral\n < 2 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[59]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[60]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[61]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[62]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[63]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[64]" [label="VariableDeclarator\n < e >", fillcolor=white   style=filled  ]
"List[65]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[66]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ConstCase[67]" [label="ConstCase\n < 3 >", fillcolor=white   style=filled  ]
"IntegerLiteral[68]" [label="IntegerLiteral\n < 3 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[69]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[70]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[71]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[72]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[73]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[74]" [label="VariableDeclarator\n < d >", fillcolor=white   style=filled  ]
"List[75]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[76]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"BreakStmt[77]" [label="BreakStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[79]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[78]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"DefaultCase[80]" [label="DefaultCase\n ", fillcolor=white   style=filled  ]
"VarDeclStmt[81]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[82]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[83]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[84]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[85]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[86]" [label="VariableDeclarator\n < f >", fillcolor=white   style=filled  ]
"List[87]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[88]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[89]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[90]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[91]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[92]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[93]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[94]" [label="VariableDeclarator\n < q >", fillcolor=white   style=filled  ]
"List[95]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[96]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"InstanceInitialization[6]" [label="InstanceInitialization\n < Instance >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[7]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Exit[8]" [label="Exit\n ", fillcolor=white   style=filled  ]
"InstanceInitialization[14]" [label="InstanceInitialization\n < Static >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[15]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Exit[16]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[23]" -> "Modifier[24]" [style=dashed, color=gray]
"Modifiers[22]" -> "List[23]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[22]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[25]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[26]" [style=dashed, color=gray]
"Modifiers[32]" -> "List[33]" [style=dashed, color=gray]
"VarDeclStmt[31]" -> "Modifiers[32]" [style=dashed, color=gray]
"VarDeclStmt[31]" -> "PrimitiveTypeAccess[34]" [style=dashed, color=gray]
"VariableDeclarator[36]" -> "List[37]" [style=dashed, color=gray]
"Opt[38]" -> "IntegerLiteral[39]" [style=dashed, color=gray]
"VariableDeclarator[36]" -> "Opt[38]" [style=dashed, color=gray]
"List[35]" -> "VariableDeclarator[36]" [style=dashed, color=gray]
"VarDeclStmt[31]" -> "List[35]" [style=dashed, color=gray]
"List[30]" -> "VarDeclStmt[31]" [style=dashed, color=gray]
"SwitchStmt[40]" -> "VarAccess[41]" [style=dashed, color=gray]
"ConstCase[44]" -> "IntegerLiteral[45]" [style=dashed, color=gray]
"List[43]" -> "ConstCase[44]" [style=dashed, color=gray]
"Modifiers[47]" -> "List[48]" [style=dashed, color=gray]
"VarDeclStmt[46]" -> "Modifiers[47]" [style=dashed, color=gray]
"VarDeclStmt[46]" -> "PrimitiveTypeAccess[49]" [style=dashed, color=gray]
"VariableDeclarator[51]" -> "List[52]" [style=dashed, color=gray]
"VariableDeclarator[51]" -> "Opt[53]" [style=dashed, color=gray]
"List[50]" -> "VariableDeclarator[51]" [style=dashed, color=gray]
"VarDeclStmt[46]" -> "List[50]" [style=dashed, color=gray]
"List[43]" -> "VarDeclStmt[46]" [style=dashed, color=gray]
"BreakStmt[54]" -> "List[56]" [style=dashed, color=gray]
"BreakStmt[54]" -> "UncheckedExceptions[55]" [style=dashed, color=gray]
"List[43]" -> "BreakStmt[54]" [style=dashed, color=gray]
"ConstCase[57]" -> "IntegerLiteral[58]" [style=dashed, color=gray]
"List[43]" -> "ConstCase[57]" [style=dashed, color=gray]
"Modifiers[60]" -> "List[61]" [style=dashed, color=gray]
"VarDeclStmt[59]" -> "Modifiers[60]" [style=dashed, color=gray]
"VarDeclStmt[59]" -> "PrimitiveTypeAccess[62]" [style=dashed, color=gray]
"VariableDeclarator[64]" -> "List[65]" [style=dashed, color=gray]
"VariableDeclarator[64]" -> "Opt[66]" [style=dashed, color=gray]
"List[63]" -> "VariableDeclarator[64]" [style=dashed, color=gray]
"VarDeclStmt[59]" -> "List[63]" [style=dashed, color=gray]
"List[43]" -> "VarDeclStmt[59]" [style=dashed, color=gray]
"ConstCase[67]" -> "IntegerLiteral[68]" [style=dashed, color=gray]
"List[43]" -> "ConstCase[67]" [style=dashed, color=gray]
"Modifiers[70]" -> "List[71]" [style=dashed, color=gray]
"VarDeclStmt[69]" -> "Modifiers[70]" [style=dashed, color=gray]
"VarDeclStmt[69]" -> "PrimitiveTypeAccess[72]" [style=dashed, color=gray]
"VariableDeclarator[74]" -> "List[75]" [style=dashed, color=gray]
"VariableDeclarator[74]" -> "Opt[76]" [style=dashed, color=gray]
"List[73]" -> "VariableDeclarator[74]" [style=dashed, color=gray]
"VarDeclStmt[69]" -> "List[73]" [style=dashed, color=gray]
"List[43]" -> "VarDeclStmt[69]" [style=dashed, color=gray]
"BreakStmt[77]" -> "List[79]" [style=dashed, color=gray]
"BreakStmt[77]" -> "UncheckedExceptions[78]" [style=dashed, color=gray]
"List[43]" -> "BreakStmt[77]" [style=dashed, color=gray]
"List[43]" -> "DefaultCase[80]" [style=dashed, color=gray]
"Modifiers[82]" -> "List[83]" [style=dashed, color=gray]
"VarDeclStmt[81]" -> "Modifiers[82]" [style=dashed, color=gray]
"VarDeclStmt[81]" -> "PrimitiveTypeAccess[84]" [style=dashed, color=gray]
"VariableDeclarator[86]" -> "List[87]" [style=dashed, color=gray]
"VariableDeclarator[86]" -> "Opt[88]" [style=dashed, color=gray]
"List[85]" -> "VariableDeclarator[86]" [style=dashed, color=gray]
"VarDeclStmt[81]" -> "List[85]" [style=dashed, color=gray]
"List[43]" -> "VarDeclStmt[81]" [style=dashed, color=gray]
"Block[42]" -> "List[43]" [style=dashed, color=gray]
"SwitchStmt[40]" -> "Block[42]" [style=dashed, color=gray]
"List[30]" -> "SwitchStmt[40]" [style=dashed, color=gray]
"Modifiers[90]" -> "List[91]" [style=dashed, color=gray]
"VarDeclStmt[89]" -> "Modifiers[90]" [style=dashed, color=gray]
"VarDeclStmt[89]" -> "PrimitiveTypeAccess[92]" [style=dashed, color=gray]
"VariableDeclarator[94]" -> "List[95]" [style=dashed, color=gray]
"VariableDeclarator[94]" -> "Opt[96]" [style=dashed, color=gray]
"List[93]" -> "VariableDeclarator[94]" [style=dashed, color=gray]
"VarDeclStmt[89]" -> "List[93]" [style=dashed, color=gray]
"List[30]" -> "VarDeclStmt[89]" [style=dashed, color=gray]
"Block[29]" -> "List[30]" [style=dashed, color=gray]
"StaticInitializer[28]" -> "Block[29]" [style=dashed, color=gray]
"List[27]" -> "StaticInitializer[28]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[27]" [style=dashed, color=gray]
"InstanceInitialization[6]" -> "Entry[7]" [style=dashed, color=gray]
"InstanceInitialization[6]" -> "Exit[8]" [style=dashed, color=gray]
"InstanceInitialization[14]" -> "Entry[15]" [style=dashed, color=gray]
"InstanceInitialization[14]" -> "Exit[16]" [style=dashed, color=gray]
"ClassDecl[5]" -> "InstanceInitialization[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "InstanceInitialization[14]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[16]" -> "VariableDeclarator[94]" [color=red, constraint=false]
"VariableDeclarator[94]" -> "Exit[16]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[94]" -> "VariableDeclarator[51]" [color=red, constraint=false]
"VariableDeclarator[94]" -> "VariableDeclarator[74]" [color=red, constraint=false]
"VariableDeclarator[94]" -> "VariableDeclarator[86]" [color=red, constraint=false]
"VariableDeclarator[51]" -> "VariableDeclarator[94]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[51]" -> "ConstCase[44]" [color=red, constraint=false]
"VariableDeclarator[74]" -> "VariableDeclarator[94]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[74]" -> "ConstCase[67]" [color=red, constraint=false]
"VariableDeclarator[86]" -> "VariableDeclarator[94]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[86]" -> "DefaultCase[80]" [color=red, constraint=false]
"DefaultCase[80]" -> "VariableDeclarator[86]" [color=blue, constraint=false, xlabel=" "] 
"DefaultCase[80]" -> "ConstCase[67]" [color=red, constraint=false]
"ConstCase[67]" -> "VariableDeclarator[74]" [color=blue, constraint=false, xlabel="T "] 
"ConstCase[67]" -> "DefaultCase[80]" [color=blue, constraint=false, xlabel="F "] 
"ConstCase[67]" -> "ConstCase[57]" [color=red, constraint=false]
"ConstCase[67]" -> "VariableDeclarator[64]" [color=red, constraint=false]
"VariableDeclarator[64]" -> "ConstCase[67]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[64]" -> "ConstCase[57]" [color=red, constraint=false]
"ConstCase[57]" -> "VariableDeclarator[64]" [color=blue, constraint=false, xlabel="T "] 
"ConstCase[57]" -> "ConstCase[67]" [color=blue, constraint=false, xlabel="F "] 
"ConstCase[57]" -> "ConstCase[44]" [color=red, constraint=false]
"ConstCase[44]" -> "VariableDeclarator[51]" [color=blue, constraint=false, xlabel="T "] 
"ConstCase[44]" -> "ConstCase[57]" [color=blue, constraint=false, xlabel="F "] 
"ConstCase[44]" -> "VarAccess[41]" [color=red, constraint=false]
"VarAccess[41]" -> "ConstCase[44]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[41]" -> "VariableDeclarator[36]" [color=red, constraint=false]
"VariableDeclarator[36]" -> "VarAccess[41]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[36]" -> "IntegerLiteral[39]" [color=red, constraint=false]
"IntegerLiteral[39]" -> "VariableDeclarator[36]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[39]" -> "Entry[15]" [color=red, constraint=false]
"Entry[15]" -> "IntegerLiteral[39]" [color=blue, constraint=false, xlabel=" "] 
"Exit[8]" -> "Entry[7]" [color=red, constraint=false]
"Entry[7]" -> "Exit[8]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"InstanceInitialization[14]" -> "InstanceInitialization[6]" -> "Modifiers[22]" -> "Opt[25]" -> "List[26]" -> "List[27]" [style=invis]
"VarDeclStmt[31]" -> "SwitchStmt[40]" -> "VarDeclStmt[89]" [style=invis]
"Modifiers[32]" -> "PrimitiveTypeAccess[34]" -> "List[35]" [style=invis]
"List[37]" -> "Opt[38]" [style=invis]
"VarAccess[41]" -> "Block[42]" [style=invis]
"ConstCase[44]" -> "VarDeclStmt[46]" -> "BreakStmt[54]" -> "ConstCase[57]" -> "VarDeclStmt[59]" -> "ConstCase[67]" -> "VarDeclStmt[69]" -> "BreakStmt[77]" -> "DefaultCase[80]" -> "VarDeclStmt[81]" [style=invis]
"Modifiers[47]" -> "PrimitiveTypeAccess[49]" -> "List[50]" [style=invis]
"List[52]" -> "Opt[53]" [style=invis]
"Modifiers[60]" -> "PrimitiveTypeAccess[62]" -> "List[63]" [style=invis]
"List[65]" -> "Opt[66]" [style=invis]
"Modifiers[70]" -> "PrimitiveTypeAccess[72]" -> "List[73]" [style=invis]
"List[75]" -> "Opt[76]" [style=invis]
"Modifiers[82]" -> "PrimitiveTypeAccess[84]" -> "List[85]" [style=invis]
"List[87]" -> "Opt[88]" [style=invis]
"Modifiers[90]" -> "PrimitiveTypeAccess[92]" -> "List[93]" [style=invis]
"List[95]" -> "Opt[96]" [style=invis]
"Entry[7]" -> "Exit[8]" [style=invis]
"Entry[15]" -> "Exit[16]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
87", fillcolor=1]
Edges [label="Total Nr. Edges
120", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
15", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
17", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}