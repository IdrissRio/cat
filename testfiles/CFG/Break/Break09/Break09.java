public class Break09 {
  static {
    int i = 0;
    switch (i) {
    case 1:
      int c;
      break;
    case 2:
      int e;
    case 3:
      int d;
      break;
    default:
      int f;
    }
    int q;
  }
}