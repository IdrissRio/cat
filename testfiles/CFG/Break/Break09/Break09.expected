NodeName: Entry[15] Successors: [IntegerLiteral[39]]
NodeName: IntegerLiteral[39] Successors: [VariableDeclarator[36]]
NodeName: VariableDeclarator[36] Successors: [VarAccess[41]]
NodeName: VarAccess[41] Successors: [ConstCase[44]]
NodeName: ConstCase[44] Successors: [VariableDeclarator[51], ConstCase[57]]
NodeName: VariableDeclarator[51] Successors: [VariableDeclarator[94]]
NodeName: VariableDeclarator[94] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: ConstCase[57] Successors: [VariableDeclarator[64], ConstCase[67]]
NodeName: VariableDeclarator[64] Successors: [ConstCase[67]]
NodeName: ConstCase[67] Successors: [VariableDeclarator[74], DefaultCase[80]]
NodeName: VariableDeclarator[74] Successors: [VariableDeclarator[94]]
NodeName: DefaultCase[80] Successors: [VariableDeclarator[86]]
NodeName: VariableDeclarator[86] Successors: [VariableDeclarator[94]]
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []

NodeName: Exit[16] Predecessor: [VariableDeclarator[94]]
NodeName: VariableDeclarator[94] Predecessor: [VariableDeclarator[51], VariableDeclarator[74], VariableDeclarator[86]]
NodeName: VariableDeclarator[51] Predecessor: [ConstCase[44]]
NodeName: ConstCase[44] Predecessor: [VarAccess[41]]
NodeName: VarAccess[41] Predecessor: [VariableDeclarator[36]]
NodeName: VariableDeclarator[36] Predecessor: [IntegerLiteral[39]]
NodeName: IntegerLiteral[39] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: VariableDeclarator[74] Predecessor: [ConstCase[67]]
NodeName: ConstCase[67] Predecessor: [ConstCase[57], VariableDeclarator[64]]
NodeName: ConstCase[57] Predecessor: [ConstCase[44]]
NodeName: VariableDeclarator[64] Predecessor: [ConstCase[57]]
NodeName: VariableDeclarator[86] Predecessor: [DefaultCase[80]]
NodeName: DefaultCase[80] Predecessor: [ConstCase[67]]
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
